safe-global/safe-wallet-web
# Fix styles on import/export widget in the safe list

Change the DataWidget component to match the redesign styles:
https://www.figma.com/file/WSpulDIM1EoSCI6Ph9QdWq/Seedless-onboarding?type=design&node-id=1548-40904&mode=design&t=zuanIiNhaqFK4G7z-0

Also removes the no longer used SafeListDrawer component.


## How to test it
- go to `/welcome/accounts` or open the sidebar on the dashboard
- see the redesign shown below

## Screenshots
<img width="484" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/ef202de7-73db-487b-a54b-a88b1ba3998a">
